### DISCUSSION TITLE
Add note field for at "Requestable Assets".

### DISCUSSION THREAD

**Discussion by Dennis-MSI** [snipe/snipe-it#11307] [Ideas]
Hi Snipe-it admins,

We would like to see an option we're the Manager can add text in a seperate field to tell us which user is going to use the asset.
Because when the manager now clicks on the button "Request" a mail will be imediatly send without info which user gets the Asset. (see screenshot)

Is that possible?

![image](https://user-images.githubusercontent.com/96120758/173572091-1b98072c-6a92-46c6-a236-be28c1899aa2.png)

**Comment by Delirium-Sama** [comment]
normally a user choosing an asset we see his name no ?

**Comment by Dennis-MSI** [comment]
That right! but what i would to see something like this:
- Manager of user requests an Asset.
- We get a mail that the manager has requested the asset, but now we do not see for which user the Asset has been requested.
